float32[16] distancia

